xero/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .api import Xero
 
-__version__ = "0.8.1"
+__version__ = "0.8.2"
```

Comment:
Can you please remove this change? The version will be changed on the next release.